test.properties

```diff
@@ -8567,6 +8567,49 @@
     test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][translation]=\
         (testray.main.component.name ~ "Translations Management")
 
+    #
+    # UM
+    #
+
+    test.batch.class.names.includes[um]=\
+        **/account/**/*Test.java,\
+        **/address-test/**/*Test.java,\
+        **/company-test/**/*Test.java,\
+        **/configuration-admin/**/*Test.java,\
+        **/expando-test/**/*Test.java,\
```

Comment:
@gaborlovasdotcom I believe these belong to Core/PEDS

Reply:
Can you confirm @vicnate5?